### DISCUSSION TITLE
DO BETTER WITH CREATNG NEW LOGINS!!!

### DISCUSSION THREAD

**Discussion by xbwxhammer** [community/community#124486] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Took me an hour to create a login! your puzzles were so INFURIATING! Then the password I typed in wasnt the one your system put in when I created the login so I had to reset the password then go through 5 different verifications to do that and finally log in!! Holy hell!!! If I dont have any projects or anything just let me login what is with all of this gatekeeping? The website is already hard enough to maneuver the UI here is SOOO bad!  Everywhere else I was searchuing everyone having this same issue!

**Comment by queenofcorgis** [comment]
Thank you for this feedback. It has been logged internally. Our team will evaluate the feedback, but we cannot guarantee a resolution or a timeline.

Any additional members who have the same request, please comment to share use cases or upvote it.

We appreciate you contributing and continuing to support other users in the GitHub Community Discussions.
